app/components/page-components/Chatbot/Commands/ChatbotDefaultCommands.vue.ts

```diff
@@ -12,6 +12,8 @@ import CollapsibleSection from 'components/shared/CollapsibleSection.vue';
 export default class ChatbotDefaultCommands extends ChatbotBase {
   searchQuery = '';
 
+  v1CommandSlugs = ['commands', 'link-protection', 'giveaway'];
```

Comment:
For better Typecheck for array with limited values, it's good to declare an item type
```typescript
 type TCommandSlug: 'commands', 'link-protection', 'giveaway';
 v1CommandSlugs: TCommandSlug[] = ['commands', 'link-protection', 'giveaway'];
```